pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1193,10 +1192,9 @@ self: super: {
   });
 
   # Remove unecessary constraint:
-  # https://github.com/agrafix/superbuffer/pull/2
-  superbuffer = overrideCabal super.superbuffer (drv: {
+  data-accessor-template = overrideCabal super.data-accessor-template (drv: {
     postPatch = ''
-      sed -i 's#QuickCheck < 2.10#QuickCheck < 2.13#' superbuffer.cabal
+      sed -i 's#template-haskell >=2.11 && <2.15#template-haskell#' data-accessor-template.cabal
```

Comment:
Could you add a comment on this line explaining why it is needed, and when we can expect to remove it?  And ideally report this upstream and link to the issue here?

Reply:
Done, thank you.